src/js/components/intro.js

```diff
@@ -0,0 +1,33 @@
+import { newText, newMultimedia, newContainer } from './helpers';
+
+/**
+ * @param {Array [objectImages]} imagesArrays es un array de imagenes de la data
```

Comment:
Usually, comments are used to *briefly* explain what something does, and the expected outcome. The process is documented if necessary. The idea is to make code that explains itself, and compliment some complex/weird behaviors with comment.

In this case, the code is clear enough to understand what its making, but comments are always welcome. The comment is longer than the code itself. Maybe you can summarize the behavior like:
```
This function receives an array of object and return a list of images
```
(Not the best example of a comment, but I hope you get the point) 
😄